cmd/skopeo/utils.go

```diff
@@ -200,6 +192,22 @@ func (opts *imageDestOptions) newSystemContext() (*types.SystemContext, error) {
 	return ctx, err
 }
 
+func (opts *globalOptions) newSystemContext() *types.SystemContext {
+	ctx := &types.SystemContext{
+		RegistriesDirPath:        opts.registriesDirPath,
+		ArchitectureChoice:       opts.overrideArch,
+		OSChoice:                 opts.overrideOS,
+		VariantChoice:            opts.overrideVariant,
+		SystemRegistriesConfPath: opts.registriesConfPath,
+		BigFilesTemporaryDir:     opts.tmpDir,
+	}
+	// DEPRECATED: We support this for backward compatibility, but override it if a per-image flag is provided.
```

Comment:
This should be removed from `imageOptions.newSystemContext` now (and I guess replaced with a comment that explains that the `imageOptions` option overrides the `global` one if both are present.

Reply:
(Non-blocking: I meant a comment specific to `--tls-verify`. I guess the current generic one works.)